parse/table_definitions_arbitrum/balancer/V2_StablePool_event_AmpUpdateStopped.json

```diff
@@ -0,0 +1,32 @@
+{
+    "parser": {
+        "abi": {
+            "anonymous": false,
+            "inputs": [
+                {
+                    "indexed": false,
+                    "internalType": "uint256",
+                    "name": "currentValue",
+                    "type": "uint256"
+                }
+            ],
+            "name": "AmpUpdateStopped",
+            "type": "event"
+        },
+        "contract_address": "SELECT pool FROM ref('V2_StablePoolFactory_event_PoolCreated')",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "balancer",
+        "schema": [
+            {
+                "description": "",
+                "name": "currentValue",
+                "type": "STRING"
+            }
+        ],
+        "table_description": "",
+        "table_name": "V2_MetaStablePool_event_AmpUpdateStopped"
```

Comment:
Should match the file name

Reply:
@medvedev1088 Corrected and went through other files to correct same mistake.